src/styles/index.scss

```diff
@@ -1,3 +1,13 @@
+@import "../styles/card.scss";
+@import "../styles/stars.scss";
+
+// * {
+//   margin: 0;
+// }
+
```

Comment:
do not leave comments in PR

Reply:
can you check, pls? i want to 'approved these changes'